terraform/environments/digital-prison-reporting/application_variables.json

```diff
@@ -82,6 +82,12 @@
       "enable_dms_replication_task": true,
       "enable_domain_builder_lambda": true,
       "enable_dbuilder_flyway_lambda": true,
+      "enable_s3_file_transfer_lambda": true,
+      "enable_s3_file_transfer_lambda_trigger": true,
+      "scheduled_s3_file_transfer_lambda_retention_days": 2,
+      "scheduled_s3_file_transfer_lambda_schedule": "cron(0 0/3 ? * * *)",
```

Comment:
After the initial data ingestion, a scheduled version of the file transfer lambda runs every 3 hours and only deletes files older than 2 days.